cmd/drone-server/server.go

```diff
@@ -169,6 +169,12 @@ var flags = []cli.Flag{
 		Usage:  "database driver configuration string",
 		Value:  "drone.sqlite",
 	},
+	cli.StringFlag{
+		EnvVar: "DRONE_PROMETHEUS_TOKEN",
```

Comment:
Would it be more descriptive to call this a Prometheus auth token?

Reply:
We already call it exactly like that within the autoscaler, so I would not rename that.

Reply:
@bradrydzewski d'oh, might be worth thinking about aliasing this in autoscaler and changing the docs to include the explicit `AUTH` bit.

We could probably do the same here.